releases/v1.1.md

```diff
@@ -0,0 +1,108 @@
+---
+title: What&#39;s New in v1.1
+toc: false
+summary: Additions and changes in CockroachDB version v1.1
+---
+
+## Oct 12, 2017
+
+With the release of CockroachDB v1.1, we’ve included support for faster migrations from other databases for developers, simplified cluster management for operators, and improved performance in real-world environments. Read more about these changes in the [v1.1 blog post](https://www.cockroachlabs.com/blog/cockroachdb-1dot1/), check out a [summary of new SQL, CLI, and Admin UI features](#summary) below, and then [upgrade to CockroachDB v1.1](../v1.1/upgrade-cockroach-version.html).
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+## Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1.0.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1.0.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1.0.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1.0.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+## Summary
+
+This section summarizes the most significant, user-facing changes included in v1.1. For a complete list of changes, including bug fixes and performance improvements, see the [release notes](index.html#testing-releases) for previous testing releases.
```

Comment:
We need to call out all the backwards-incompatible changes here too. (I'd put this before any discussion of new features). We should also call out somewhere that upgrading from 1.0.x to 1.1 requires a different procedure from upgrades within the 1.0.x series.

Reply:
@bdarnell, thank you for calling that out! I've added a list of backwards-incompatible changes and note that the upgrade process is different.